LICENSE.md

```diff
@@ -0,0 +1,10 @@
+The MIT License (MIT)
+--
+
+Copyright © 2022 <copyright holders>
```

Comment:
Not sure if this started a few years back or what, but you probably want a range + your name here.
```suggestion
Copyright © 2016-present <copyright holders>
```

Your initial commit was in 2016 so I based it on that.
```
commit 533b7c4b64d07d5ebb6080f80721d70254e8b2ce
Date:   Sat Apr 30 17:11:56 2016 +0200
```